src/runtime/audience-action-flow.ts

```diff
@@ -321,6 +330,69 @@ export class AudienceActionFlow {
   }
 
   /**
+   * Populates localStorage with PPS configuration parameters based on
+   * SurveyDataTransferRequest.
+   * @param {SurveyDataTransferRequest} request
+   * @return {boolean}
+   * @private
+   **/
+  async configureAnswerPpsData(request: SurveyDataTransferRequest) {
+    const iabAudienceKey = 'iabtaxonomiesvalues';
+    // PPS value field is optional and category may not be populated
+    // in accordance to IAB taxonomies.
+    const ppsConfigParams = request
+      .getSurveyQuestionsList()!
+      .filter(
+        (question) => question.getSurveyAnswersList()![0].getPpsValue() !== null
+      )
+      .map((question) => question.getSurveyAnswersList()![0].getPpsValue());
+
+    if (ppsConfigParams.length === 0) {
+      // Answers with no PPS parameters should always evaluate to true with
+      // the feature flag enabled.
+      return true;
+    }
+
+    const existingIabTaxonomy = await this.storage_.get(
+      iabAudienceKey,
+      /* useLocalStorage= */ true
+    );
+    let existingIabTaxonomyMap;
+    if (!existingIabTaxonomy) {
+      existingIabTaxonomyMap = {
+        '[googletag.enums.Taxonomy.IAB_AUDIENCE_1_1]': ppsConfigParams,
```

Comment:
yes, since the format takes on: 
```
googletag.setConfig({   pps: {     taxonomies: {      [googletag.enums.Taxonomy.IAB_AUDIENCE_1_1]:         {values: ['12', '13', '14']},      [googletag.enums.Taxonomy.IAB_CONTENT_2_2]:        {values: ['22', '25', '57']},    }  } });
--
```
i updated the key to accommodate the key for the values as defined the API.